resources/definitions/blocks_pros30.def.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": 2,
+  "name": "Blocks Pro S30",
+  "inherits": "blocks_base",
+  "metadata": {
+    "visible": true,
+
+    "quality_definition": "blocks_base",
+
+    "platform": "blocks_pros30_platform.stl",
+
+    "machine_extruder_trains": {
+      "0": "blocks_pros30_extruder_0"
+
+      
+    }
+  },
+  "overrides": {
+    "machine_name": { "default_value": "Blocks Pro S30" },
+
+    "machine_width": { "default_value": 310 },
+    "machine_height": { "default_value": 290 },
+    "machine_depth": { "default_value": 310 },
+
+    "retraction_retract_speed": { "value": 40, "maximum_value_warning": 130 },

```

Comment:
Why not just define the maximum value_warning in the base file? THey seem to be the same for all machines.